client/src/components/ItemDetail/ItemDetail.tsx

```diff
@@ -81,34 +119,71 @@ function ItemDetail({
       >
         <CloseIcon />
       </IconButton>
-      <div style={{ width: "100vw", height: "60vh", overflow: "hidden" }}>
-        <img
-          src={item.media ? item.media : PLACEHOLDER_ITEM_MEDIA}
-          style={{ height: "100%", width: "100%", objectFit: "cover" }}
-        />
-      </div>
+      <div
+        style={{
+          width: "100vw",
+          height: `${Math.floor((1 - footerPercentageHeight) * 100)}vh`,
+          overflow: "hidden",
+          backgroundPosition: "center",
+          backgroundSize: "cover",
+          backgroundImage: `url('${
+            item.media ? item.media : PLACEHOLDER_ITEM_MEDIA
+          }')`,
+        }}
+      />
       <div style={{ width: "100vw" }}>
         {/*TODO(#198): zIndex layering helper class needed to manage layers on page*/}
         <div
           style={{
             position: "absolute",
             zIndex: 1003,
             width: "100%",
-            height: `${cornerRoundingPercentage}vw`,
-            marginTop: `-${cornerRoundingPercentage}vw`,
-            borderTopLeftRadius: `${cornerRoundingPercentage}vw`,
-            borderTopRightRadius: `${cornerRoundingPercentage}vw`,
+            height: `${cornerRoundingRadius}`,
+            marginTop: `-${cornerRoundingRadius}`,
+            borderTopLeftRadius: `${cornerRoundingRadius}`,
+            borderTopRightRadius: `${cornerRoundingRadius}`,
             backgroundColor: "#FFFFFF",
+            textAlign: "center",
           }}
-        />
+        >
+          {/*Floating Expand button*/}
+          {isOverflow && (
+            <Button
+              fullWidth={true}
+              style={{ height: `${cornerRoundingRadius}` }}
+              onClick={() => setMediaCollapsed(!mediaCollapsed)}
+            >
+              {mediaCollapsed ? "Show Less" : "Show More"}
+              {mediaCollapsed ? <ExpandMoreIcon /> : <ExpandLessIcon />}
+            </Button>
+          )}
+        </div>
+        {/*Content Div Container*/}
```

Comment:
nit: remove {}?

Reply:
surrounding the comment? Not having {} makes the compiler complain